### PR TITLE
[OGWA-12] As a non-registered user, I want to input my data, so that I can register as a user

### PR DESCRIPTION
**Pull request by rifkyrizkita**
As a non-registered user, I want to input my data, so that I can register as a user

### CODE REVIEW

File: projects/client/src/pages/Register.jsx
Comment (asc33nzio) [inline projects/client/src/pages/Register.jsx:1]: camel case pls brader

File: projects/server/src/config/config.json
Comment (asc33nzio) [inline projects/server/src/config/config.json:1]: ini udah jadi config.js harusnya dan dotenv

**Review by asc33nzio — CHANGES_REQUESTED** [review]
a few conflicts and rollback, fix required

File: projects/client/src/views/Register/components/registerField.jsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+      .oneOf([Yup.ref("password")], "Passwords must match"),
+  });
+
+  const handleSubmit = async (data) => {
+    try {
+      const response = await Axios.post(
+        "http://localhost:8000/api/users/register",
+        data
+      );
+      console.log(response);
```
Comment (gangsararyop) [inline projects/client/src/views/Register/components/registerField.jsx:40]: hapus ya

File: projects/client/src/views/Register/components/registerField.jsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      toast.error(error.response.data.message, {
+        position: "top-right",
+        autoClose: 2000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+        theme: "dark",
+    });
```
Comment (gangsararyop) [inline projects/client/src/views/Register/components/registerField.jsx:64]: kalo udh ada toast ga perlu pake console.log

**Pull request closed without merging**